src/xenia/kernel/xboxkrnl/xboxkrnl_video.cc

```diff
@@ -30,6 +30,13 @@ UPDATE_from_uint32(kernel_display_gamma_type, 2020, 12, 31, 13, 1);
 DEFINE_double(kernel_display_gamma_power, 2.22222233,
               "Display gamma to use with kernel_display_gamma_type 3.",
               "Kernel");
+DEFINE_bool(kernel_display_interlaced, false,
+            "Whether to use interlaced display mode in games.", "Kernel");
```

Comment:
Why would you want interlaced?

Reply:
> Why would you want interlaced?

Because it's a technique used in older video formats to improve picture quality.
I added it as an option in case someone needs to use it for legacy purposes.

Reply:
I think it could be beneficial for some.

Reply:
Interlaced on a progressive display (anything made within the last two decades or so) makes the output look worse, not better: https://en.wikipedia.org/wiki/Deinterlacing#/media/File:Interlaced_video_frame_(car_wheel).jpg

> I think it could be beneficial for some.

How? If interlacing was at all desirable PCSX2 wouldn't need **de**interlacing.
Even if you're outputting to a CRT somehow and need interlaced output Windows does that for you.

Reply:
> Interlaced on a progressive display (anything made within the last two decades or so) makes the output look worse, not better: https://en.wikipedia.org/wiki/Deinterlacing#/media/File:Interlaced_video_frame_(car_wheel).jpg
> 
> How? If interlacing was at all desirable PCSX2 wouldn't need **de**interlacing. Even if you're outputting to a CRT somehow and need interlaced output Windows does that for you.

Good point. However, there are some legacy systems and older formats that still use interlaced video. Having the option to output interlaced video can be beneficial.

Reply:
> > Interlaced on a progressive display (anything made within the last two decades or so) makes the output look worse, not better: [en.wikipedia.org/wiki/Deinterlacing#/media/File:Interlaced_video_frame_(car_wheel).jpg](https://en.wikipedia.org/wiki/Deinterlacing#/media/File:Interlaced_video_frame_(car_wheel).jpg)
> > How? If interlacing was at all desirable PCSX2 wouldn't need **de**interlacing. Even if you're outputting to a CRT somehow and need interlaced output Windows does that for you.
> 
> Good point. However, there are some legacy systems and older formats that still use interlaced video. Having the option to output interlaced video can be beneficial.

Describe a situation where this would be beneficial.

Reply:
> Describe a situation where this would be beneficial.

Interlaced video can still be beneficial in situations where you need to work with legacy systems and older formats.
For example, older game consoles like Sega Genesis or Super Nintendo use interlaced video, so having the option to output interlaced video could help when working with legacy systems.